packages/sdk/src/modules/loggerModule/index.ts

```diff
@@ -0,0 +1,16 @@
+import {
+  LoggerFactory,
+  LoggerOptions,
+  LoggerType,
+} from "@vue-storefront/logger";
+import type { Module } from "../../types";
+
+export const loggerModule = (options?: LoggerOptions) => {
```

Comment:
Do not get logger options directly from the logger itself, loggerOptions are not necessary exactly the same options that we might want to expose via sdk module options, this is just a coincidence at this stage. And also, changing the logger options might break code that is using the module but not necessary the module. If I add some server related options to the logger it will be reflected in the module but it does not make any sense. We need to have separate options type for the module.

Reply:
Ok that makes sense. I'll add one

Reply:
Changed to what we've discussed.